lms/djangoapps/course_api/blocks/serializers.py

```diff
@@ -52,6 +53,7 @@ def __init__(
     SupportedFieldType('due'),
     SupportedFieldType('contains_gated_content'),
     SupportedFieldType('has_score'),
+    SupportedFieldType(DiscussionsTransformer.IS_DISCUSSABLE, DiscussionsTransformer),
```

Comment:
Is there any advantage to creating a separate transformer for this? We can just fetch it using the regular machinery by adding it as a supported type. 
i.e. this single line should be all that's needed in this entire PR. 
```suggestion
    SupportedFieldType('is_discussable'),
```

Reply:
@xitij2000 

It was a little more complicated that simply adding the field. By default, the field is not available on the blocks in the `block_structure`, so you have to request it. This is done in `BlocksAPITransformer` where the field has been added to the basic set that are requested. In order to invalidate the data available for a particular block structure and thus show this field in the API response, you also have to increment `WRITE_VERSION` and `READ_VERSION`. All of this is included in the most recent commit